.github/CONTRIBUTING.md

```diff
@@ -2,113 +2,14 @@
 
 ## Web Server
 
-### Simple PHP Server
+### Docker Web Server with Nginx and PHP
 
 For a simple local development environment running on PHP, you will need:
-
```

Comment:
I think it's worth keeping the manual installation instructions around in case people want to do a setup without docker. Can we move them to a "manual setup" section, and have the Docker instructions under a new "recommended setup" section?

Reply:
That said, after reviewing the instructions here, it looks like they need updating a bit :sweat_smile:

Reply:
Cool I added these two sessions and update the recommended versions of php and nodejs